### ISSUE TITLE
Stray WPS513 warning.

### ISSUE THREAD

**Issue by agatti** [wemake-services/wemake-python-styleguide#1632]
# Bug report

`WPS513` is emitted on `if`/`else`/`if` chains that contain other statements in the last `if` block.

## What's wrong

The following snippet:

```python
if private_key is not None:
    if isinstance(private_key, PKey):
        key = private_key
    else:
        if isinstance(private_key, str):  # WPS513 Found implicit `elif` condition
            stream = io.StringIO()
            stream.write(private_key)
            stream.seek(0)
            private_key = stream
        key = RSAKey.from_private_key(private_key)
```

triggers a stray WPS513 warning.  Seems like the assignment to `key` is not taken into account.

## How is that should be

`WPS513` should only be emitted if the dangling `if` statement is the only one present in the `else` block.

**Comment by Jrryy** [comment]
You are right. Adding the following template to the 513 tests, it throws an incorrect violation:

```python
if some:
    ...
else:
    if other:
        ...
    ...
```

If it is no problem, I will be taking a look and fixing this violation for such cases.

**Comment by sobolevn** [comment]
@Jrryy will you please take this over?

**Comment by Jrryy** [comment]
Sure, already at it

**Comment by sobolevn** [comment]
Thanks!
